### ISSUE TITLE
PHP Warning:  Invalid argument supplied for foreach()

### ISSUE THREAD

**Issue by AiratHalitov** [buddyboss/buddyboss-platform#1380]
**Describe the bug**
LOG:
```
[php7:warn] PHP Warning:  Invalid argument supplied for foreach() in /site_path/wp-content/plugins/buddyboss-platform/bp-xprofile/bp-xprofile-functions.php on line 1696
```

Source Code: https://github.com/buddyboss/buddyboss-platform/blob/dev/src/bp-xprofile/bp-xprofile-functions.php#L1696

buddyboss-platform v1.4.9
WordPress 5.5
PHP 7.3.21

**Comment by AiratHalitov** [comment]
Now it is on line 1700.
In v1.5.2 the bug is still actual

**Comment by janakkaneriya007** [comment]
@AiratHalitov Did you check my patch? I am not able to reproduce on my end. Do you know how to vardump variable? Can you please show what values you are getting there?

**Comment by AiratHalitov** [comment]
> Did you check my patch?

No. I will check it now. 

> Do you know how to vardump variable? Can you please show what values you are getting there?

`var_dump($original_option_values);` in line 1700 (before foreach) with your patch:

```
array(8) { ["vk"]=> string(27) "https://vk.com/username" ["instagram"]=> string(39) "https://www.instagram.com/username/" ["pinterest"]=> string(39) "https://www.pinterest.com/username/" ["youTube"]=> string(61) "https://www.youtube.com/user/username/?sub_confirmation=1" ["facebook"]=> string(37) "https://www.facebook.com/username" ["twitter"]=> string(0) "" ["whatsapp"]=> string(0) "" ["telegram"]=> string(0) "" }
```

**Comment by janakkaneriya007** [comment]
@AiratHalitov Dumped values look fine, You may not face this issue with given patch. Let me know if you still face the issue.

**Comment by sourabhmatolia** [comment]
@AiratHalitov 

Issue created above is released in Buddyboss Platform v1.5.3.

Please find release notes here:
https://www.buddyboss.com/resources/buddyboss-platform-releases/1-5-3/

Please update the plugin to see the latest changes.

Thank you.
